modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -19,90 +20,251 @@ To configure a connector in Anypoint Studio:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The unit of time for the timeout property. |  |  
+|===
+. Configure the following settings in the *Advanced* tab:
 +
-. Based on the operation what you dragged to the canvas, configure the following fields:
-.. Consumer trigger:
-** Topic - The name of the Kafka topic where to consume messages.
-** Partition offsets (Optional) - list of offsets for configuring partitions. For each element in the list, specify the partition index and offset.
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Operation Timeout a| Number |  |  | 
+| Operation Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  |  | 
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Configure to use repeatable streams. |  | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable. |  `#[payload]` | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
+
+== Configure the Publish Operation
+
+. Drag the *Publish* operation to the Studio canvas.
+. Configure the *Publish* operation in the *General* tab:
 +
-image::kafka/kafka-consumer-studio-config.png[]
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Topic a| String |  The topic to publish to. |  | 
+| Partition a| Number |  (Optional) The topic partition.  |  | 
+| Key a| Binary |  (Optional) Key for the published message. |  | 
+| Message a| Binary |  (Optional) Message content of the message. |  `#[payload]` | 
+| Headers a| Object |  (Optional) Headers for the message. |  |  
+|===
 +
-.. Producer operation:
-** Topic - Topic to send the message to.
-** Key - Key belonging to the message that is going to be sent.
-** Message - Message to be sent.
+. Configure the following settings in the *Advanced* tab:
 +
-image::kafka/kafka-producer-studio-config.png[]
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Transactional Action a| Enumeration, one of:
 
-== Configure an Input Source
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  `JOIN_IF_POSSIBLE` | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable. |  `#[payload]` | 
```

Comment:
I know we have this wording all over, but since this is in a User Guide, I would improve it:

An expression that evaluates the operation's output. The outcome is stored in the target variable.